src/pykeen/models/inductive/inductive_nodepiece.py

```diff
@@ -14,7 +14,7 @@
     DistMultInteraction,
     EmbeddingSpecification,
     Interaction,
-    NodePieceRepresentation,
```

Comment:
I think i'd actually prefer it to be just `Representation` since it's shorter and more to the point

or is it finally time to change to `Encoder`?

Reply:
I'd prefer `Representation` as I would regard an encoder something which maps representations to representations rather than indices to representations.

Reply:
done in bdbdc62 and 4b31b0c